test/helpers/secrets.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+/**
+ * A helper function to get secrets needed by tests
+ */
+function getTestSecret(secretName) {
+  const envVar = process.env[secretName]
+  return envVar
+}
+
+/**
+ * Checks whether any of the secrets needed by the test are missing or if the
+ * FORCE_RUN_TESTS_WITH_SECRETS env var is set. This is set by the github
+ * actions workflow when tests are being run on the newrelic repository where
+ * secrets should be available. In this case, even if the license key that a
+ * test is looking for is not present, run the test anyway and let it fail,
+ * because it should be present on the newrelic repo.
+ */
+function shouldSkipTest() {
+  const secrets = Array.from(arguments)
```

Comment:
Not sure I understand the need for this code? Won't there only ever be a single value as we are grabbing from a single ENV var? Why do we need a generic "arguments"?

Beyond that comment, you could use `...args` if you didn't want to use `Array.from()`

Reply:
Ya, just an attempt at future-proofing. But ya, `...args` is convention, I'll switch that up.